src/acceptance-test/java/uk/gov/ida/verifyserviceprovider/builders/V2MatchingDataset.java

```diff
@@ -0,0 +1,38 @@
+package uk.gov.ida.verifyserviceprovider.builders;
```

Comment:
This class isn't a builder and doesn't call itself one, so it probably shouldn't be in this package.  I'll probably need to turn it into a builder in my story though, so I'll rename it then.